tests/python/gaia-ui-tests/gaiatest/apps/base.py

```diff
@@ -121,6 +122,15 @@ def keyboard(self):
         from gaiatest.apps.keyboard.app import Keyboard
         return Keyboard(self.marionette)
 
+    @property
+    def manifest_url(self):
+        return '{}{}{}/manifest.webapp'.format(self.DEFAULT_PROTOCOL, format(self.__class__.__name__).lower(), self.DEFAULT_APP_HOSTNAME)
```

Comment:
Nit:  `format(self.__class__.__name__)` can be replaced by  `self.__class__.__name__` as we're already in a format and `self.__class__.__name__` should already returns a string